library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -235,6 +241,7 @@
   private int sampleBytesWritten;
   private boolean sampleRead;
   private boolean sampleSeenReferenceBlock;
+  private Inflater sampleInflator;
```

Comment:
isn't this a per-track setting? i.e. a file can have one compressed track and one non-compressed track right?